packages/experimental-nextjs-app-support/README.md

```diff
@@ -204,7 +204,7 @@ That way, if you repeat the query in your Client Component using `useSuspenseQue
 
 #### Usage with `useReadQuery`.
 
-You can also use this approach in combination with `useReadQuery` in Client Components. Use the render prop approach to get a `QueryRef` that you can pass to your Client Component:
+Just like using `useBackgroundQuery` and `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Comment:
```suggestion
Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Changing this to `with` makes this sound a bit more like they work together.

Reply:
I think the thing I'm missing here is the "what" that makes `PreloadQuery` similar to `useBackgroundQuery`. I keep expecting to read something like "Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` _to avoid request waterfalls_". Adding a "to x..." statement I think would help make it obvious what the connection between `PreloadQuery` and `useBackgroundQuery` is (obviously feel free to add what you were intending. Your point might not be about avoiding waterfalls).